lib/langchain/llm/google_palm.rb

```diff
@@ -75,20 +75,20 @@ def complete(prompt:, **params)
     #
     # Generate a chat completion for a given prompt
     #
-    # @param prompt [HumanMessage] The prompt to generate a chat completion for
+    # @param prompt [String] The prompt to generate a chat completion for
     # @param messages [Array<AIMessage|HumanMessage>] The messages that have been sent in the conversation
```

Comment:
is it ok to use Message objects instead of structured Hashes for `messages` and `examples` ?

Reply:
So maybe the Message objects are only known to the Conversation class but not the LLMs. Wdyt?

I think it's weird if we use the Message objects in _some_ places in the Conversation classes and in _some_ places in the LLM::* classes. It should either be used everywhere, always, or a clean separation where when a class knows about Messages it uses them everywhere in that class but doesn't randomly expose it to other classes.